randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -388,28 +386,40 @@ Extra - map_name: rm_a5c11
       Any of the following:
           Can Use Bombs
           Missiles ≥ 10 and Can Use Power Bombs
```

Comment:
This is missing a way to do this with supers. 3 supers would probably be enough? Dunno.